src/test/java/core/basesyntax/service/impl/FruitTransactionParserServiceImplTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitReport;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.FruitTransactionParserService;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitTransactionParserServiceImplTest {
+    private FruitTransactionParserService fruitTransactionParserService =
+            new FruitTransactionParserServiceImpl();
+
+    @Before
+    public void cleanBefore() {
```

Comment:
Not fixed.
You should not clean before the test. Clean after the test
![image](https://user-images.githubusercontent.com/117547096/214878818-ea0770e7-5ba2-42ee-978c-905f3c17811a.png)